.bumpversion.cfg

```diff
@@ -1,9 +0,0 @@
-[bumpversion]
```

Comment:
@gooofy I see a bunch of changes in the PR that seem related to your fork rather than actual feature - I'm actively maintaining this package and happy to merge PR contributions, if you'd still like to have your changes merged please either remove the non-related changes or create a new PR with only the specific feature-related changes